demo/features/steps/0453-issue-credential.py

```diff
@@ -1,7 +1,5 @@
 from behave import given, when, then
 import json
-from time import sleep
```

Comment:
I'm a little bit puzzled about the fact that this demo got simpler by deleting stuff. Is it because we wrote a new demo that shows vc_di plus the stuff that's been deleted here? I would have expected this demo to get more complex as vc_di was added to the demo as well...

Reply:
I suspect the deletions are a side effect of a formatting tool, I've noticed that occasionally there are code changes due to code formatting standards changing.

Reply:
yeah i think its automatically pruning unused imports, at least that is what i suspect

Reply:
verified, at least with spot check.  resolving.